Assets/VRTK/Scripts/Interactions/VRTK_InteractGrab.cs

```diff
@@ -100,9 +104,30 @@ public GameObject GetGrabbedObject()
 
         protected virtual void Awake()
         {
-            interactTouch = GetComponent<VRTK_InteractTouch>();
-            controllerActions = GetComponent<VRTK_ControllerActions>();
-            controllerEvents = GetComponent<VRTK_ControllerEvents>();
+            if (controllerEvents == null)
+            {
+                controllerEvents = GetComponent<VRTK_ControllerEvents>();
+            }
+            if (controllerEvents == null)
+            {
+                throw new System.Exception("InteractGrab requires a non-null VRTK_ControllerEvents instance. The field controllerEvents isn't set and there was no instance found on gameObject.");
```

Comment:
Well, if this is null then other things are going to throw null exception errors. I'll defer to as to how you want it handled.